test/regression_tests.py

```diff
@@ -1,28 +1,23 @@
-import sys
 import os
-from pygit2 import Repository
+import sys
 
 # To help discover local modules
 REPO_ROOT = os.path.join(os.path.dirname(os.path.abspath(__file__)), "..")
 sys.path.append(REPO_ROOT)
 
+import datetime
+
+from ts_scripts import marsgen as mg
+from ts_scripts.api_utils import test_api
 from ts_scripts.install_from_src import install_from_src
 from ts_scripts.regression_utils import test_regression
-from ts_scripts.api_utils import test_api
-from ts_scripts import print_env_info as build_hdr_printer
 from ts_scripts.utils import check_python_version
-from ts_scripts import marsgen as mg
-
-import datetime
 
 now = datetime.datetime.now()
 print("Current date and time : " + now.strftime("%Y-%m-%d %H:%M:%S"))
 
 check_python_version()
 
-git_branch = Repository('.').head.shorthand
-build_hdr_printer.main(git_branch)
```

Comment:
are changes to this file from another PR?

Reply:
Because we are passing TS repo as a shared volume to Docker container, it was erroring on L23. I can keep it and check for the ENV variable. Should I do this?

Reply:
or we can use this opportunity to get rid of pygit2?

Reply:
No iz fine